### PR TITLE
Fixed _mm256_set_m128 is only availble on gcc8+. issue#5072

### PR DESCRIPTION
**Pull request by whyb**
Fixed _mm256_set_m128 is only availble on gcc8+. issue https://github.com/Tencent/ncnn/issues/5072

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5075) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5075) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: nihui<br/>:x: whyb<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=5075) it.</sub>

File: src/layer/x86/shufflechannel_x86.cpp [hunk 1]
```diff
@@ -345,7 +345,11 @@ int ShuffleChannel_x86::forward(const Mat& bottom_blob, Mat& top_blob, const Opt
                     __m256 _p0 = _mm256_loadu_ps(ptr0);
                     // macro `_mm256_loadu2_m128` is declared in Intel® Intrinsics Guide but somehow missed in <immintrin.h>
                     // __m256 _p1 = _mm256_loadu2_m128(ptr2, ptr1);
-                    __m256 _p1 = _mm256_set_m128(_mm_loadu_ps(ptr2), _mm_loadu_ps(ptr1));
+                    // issue #5072 _mm256_set_m128 is only availble on gcc8+
+                    // __m256 _p1 = _mm256_set_m128(_mm_loadu_ps(ptr2), _mm_loadu_ps(ptr1));
+
```
Comment (nihui) [inline src/layer/x86/shufflechannel_x86.cpp:350]: 这些废弃的注释和代码可以删掉

Comment (whyb) [reply src/layer/x86/shufflechannel_x86.cpp]: > 这些废弃的注释和代码可以删掉

Removed

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**